server/views/pages/componentRequestForm.njk

```diff
@@ -5,238 +5,265 @@
-        classes: "govuk-!-width-three-quarters"
-      },
-      errorMessage: validationErrors | findError('repo_description')
-    }) }}
-    {{ govukRadios({
-      classes: "govuk-radios--inline",
-      idPrefix: "base_template",
-      name: "base_template",
-      value: submittedForm.base_template,
-      errorMessage: validationErrors | findError('base_template'),
-      fieldset: {
-        legend: {
-          text: "Github template repository *",
-          classes: 'govuk-fieldset__legend--s'
+        id: "jira_project_keys",
+        name: "jira_project_keys",
+        value: submittedForm.jira_project_keys,
+        classes: "govuk-!-width-one-third",
+        label: {
+          text: "Jira Project Key",
+          classes: "govuk-fieldset__legend--s"
         }
-      },
-      items: [
-        {
-          value: "hmpps-template-kotlin",
-          text: "hmpps-template-kotlin"
+      }) }}
+      {{ govukInput({
+        id: "github_projects_teams_admin",
+        name: "github_projects_teams_admin",
+        value: submittedForm.github_projects_teams_admin,
+        errorMessage: validationErrors | findError('github_projects_teams_admin'),
+        classes: "govuk-!-width-two-third",
+        label: {
+          text: "Github Repository Admin Teams *",
+          classes: "govuk-fieldset__legend--s"
         },
-        {
-          value: "hmpps-template-typescript",
-          text: "hmpps-template-typescript"
-        }]
-    }) }}
-    {{ govukSelect({
-      id: "github_project_visibility",
-      name: "github_project_visibility",
-      value: submittedForm.github_project_visibility,
-      label: {
-        text: "Github project visbility *"
-      },
-      items: [
-        {
-          value: "public",
-          text: "Public"
+        hint: {
+          text: "Separated by comma eg: hmpps-developers,haha"
+        }
+      }) }}
+      {{ govukInput({
+        id: "github_project_teams_write",
+        name: "github_project_teams_write",
+        value: submittedForm.github_project_teams_write,
+        errorMessage: validationErrors | findError('github_project_teams_write'),
+        classes: "govuk-!-width-two-third",
+        label: {
+          text: "Github Repository Teams with write permissions",
+          classes: "govuk-fieldset__legend--s"
         },
-        {
-          value: "private",
-          text: "Private"
+        hint: {
+          text: "Separated by comma eg: hmpps-developers,haha"
+        }
+      }) }}
+      {{ govukInput({
+        id: "github_project_branch_protection_restricted_teams",
+        name: "github_project_branch_protection_restricted_teams",
+        value: submittedForm.github_project_branch_protection_restricted_teams,
+        errorMessage: validationErrors | findError('github_project_branch_protection_restricted_teams'),
+        classes: "govuk-!-width-two-third",
+        label: {
+          text: "Github Project Teams Branch protection restricted teams",
```

Comment:
Github project branch protection restricted teams

(That's really clunky but we don't need two 'teams'.. maybe something better?)